enhancements/network/external-lb-vips.md

```diff
@@ -0,0 +1,242 @@
+  environment before the IPI deployment.
+* We do not support the update of VIP fields in `PlatformStatus` therefore the external loadbalancer VIPs can never change.
+  This will be addressed in the future, but for TechPreview we don't support it yet.
+
+### Drawbacks
+
+This requires more work from the customer to set up the external loadbalancer and the VIPs.
+From our perspective, this takes critical infrastructure out of our control. This can complicate support of
+such environments since coordination would be required with the owner of the external loadbalancer to debug problems.
+
+## Design Details
+
+### Test Plan
+
+CI will be in charge of testing this feature.
+Platforms who want to support this will have their jobs.
+Note: vsphere already has a CI job for external loadbalancer. We'll work together to converge our tools.
+
+### Graduation Criteria
+
+The Tech-Preview target is OCP 4.13 and GA would be 4.14.
+
+
+#### Dev Preview -> Tech Preview
+
+- Ability to use an external loadbalancer from end to end without manual intervention from the customer
+  except the documented requirements.
+- Customers will pre-populate their loadbalancer backends with all the IPs available in the subnet for the
+  control-plane.
+- End user documentation, API stability
+- Sufficient test coverage
+- Gather feedback from customers
+
+#### Tech Preview -> GA
+
+- [Ability to change `apiVIP` and `ingressVIP` on a deployed cluster](https://issues.redhat.com/browse/RFE-2085)
+- [Support for Static IPs](https://github.com/openshift/enhancements/pull/1267) would be a must so
+  customers don't have to pre-populate their loadbalancer with all IPs in a subnet.
+- Migration from internal to external loadbalancer (and vice versa)
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- User facing documentation created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+#### Removing a deprecated feature
+
+NA
+
+### Upgrade / Downgrade Strategy
+
+An open question about migration from internal loadbalancer to external, but otherwise there should
+not be any upgrade impact.
+
+### Version Skew Strategy
+
+NA
+
+### Operational Aspects of API Extensions
+
+NA
+
+#### Failure Modes
+
+NA
+
+#### Support Procedures
+
+
+## Implementation History
+
+* `vsphere` already doesn't set API fields in `PlatformStatus` for external loadbalancer in IPI/UPI.
+   For backward compatibility, we'll keep it supported only on vsphere so their customers aren't broken
+   during an update.
+
+## Alternatives
+
+* Implement another internal loadbalancer like it was proposed with [BGP](https://github.com/openshift/enhancements/pull/1281). This idea was ejected for now because it didn't get much traction outside of the OpenStack team. The External LB solution was meeting our customer requirements as was a cross-platform interest so we decided to take that direction for now.
```

Comment:
```suggestion
* Implement another internal loadbalancer like it was proposed with [BGP](https://github.com/openshift/enhancements/pull/1281). This idea was rejected for now because it didn't get much traction outside of the OpenStack team. The External LB solution was meeting our customer requirements as was a cross-platform interest so we decided to take that direction for now.
```